seacatauth/external_login/authentication/providers/saml.py

```
@@ -42,25 +43,71 @@ class SamlAuthProvider(ExternalAuthProviderABC):
+		Prepare SAML2 client instance, initializing it if necessary.
+
+		Returns:
+			saml2.client.Saml2Client instance or None if initialization failed.
+		"""
+		if self.SamlClient is not None:
+			return self.SamlClient
+
+		try:
+			config = self._init_saml_config()
+			self.SamlClient = saml2.client.Saml2Client(config)
+		except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)